gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -1394,4 +1596,18 @@ public void unauthenticatedAccessToPublicBuckets_googleCloudProperties() throws
 
     assertThat(fileStatuses).isNotEmpty();
   }
+
+  private static void initializeInMemoryFileSystem(FileSystem ghfs, String initUriString)
```

Comment:
I dont see usages of this method, can we remove it ?

Reply:
Removed the method